lib/scope.js

```diff
@@ -1792,6 +1808,27 @@ module.exports = {
       await scope.setFields( scope, { var_data, ensure_navigation: false, ensure_all_vars_used: true });
     },  // Ends set_regular_var()
 
+    set_secret_var: async ( scope, var_name, set_to_env_name ) => {
+      /** Sets a non-story table variable to a "secret" value (i.e. an env var) */
+      let set_to = process.env[set_to_env_name];
+      if ( set_to === undefined ) {
+        let set_to_env_msg = `The GitHub SECRET "${ set_to_env_name }" doesn't exist. Check for a typo and check your workflow file.`
+        await scope.addToReport( scope, { type: `error`, value: set_to_env_msg })
+      }
+
+      let var_data = await scope.normalizeTable( scope, {
+              var_data: [{
+                var: var_name,
+                value: set_to,
+                trigger: scope.trigger_not_needed_flag,
```

Comment:
Non-blocking - it may be worth adding a comment that we intend to avoid letting people set secret variables in a regular story table/as part of a group of other fields, so they should never need a trigger variable.

Reply:
Never mind, this is explicitly the `set_secret_var` step. Lost my place.